roles/common/defaults/main.yml

```diff
@@ -120,6 +120,7 @@ common:
         - python-virtualenv
         - python-mysqldb
         - python-jinja2
+        - PyYAML
```

Comment:
going to remove from ubuntu list as we do not have this problem on ubuntu. Is rhel only issue.